### DISCUSSION TITLE
How to implement contentEditable with drag and drop elements ?

### DISCUSSION THREAD

**Discussion by anisulislampranto** [clauderic/dnd-kit#1135] [Q&A]
hi everyone, you can see i am trying to edit drag and drop elements but it is not working as expected. help me to find out the issue. 

https://github.com/clauderic/dnd-kit/assets/76773444/ab125a5a-425f-401b-9c89-f3a83ed3082b

**Comment by taziksh** [comment]
Hey, can you share a code snippet? It's easier to debug that way. The trick is to add the `contentEditable` prop at the right level in the component tree (which you can determine by Inspect Element).
